libs/composer/vendor/composer/autoload_classmap.php

```diff
@@ -4246,6 +4246,7 @@
     'ilExerciseManagementGUI' => $baseDir . '/../../Modules/Exercise/classes/class.ilExerciseManagementGUI.php',
     'ilExerciseMemberTableGUI' => $baseDir . '/../../Modules/Exercise/classes/class.ilExerciseMemberTableGUI.php',
     'ilExerciseMembers' => $baseDir . '/../../Modules/Exercise/classes/class.ilExerciseMembers.php',
+    'ilExerciseMembersFilter' => $baseDir . '/../../Modules/Exercise/classes/class.ilExerciseMembersFilter.php',
```

Comment:
This doesn't belong here, right?

Reply:
Seems to be forgotten in a prior commit by another dev. I always rebuild the autoload classmap when I introduce new files, which I have done here. But of course my new files are completely irrelevant for the classmap because they are located under `./src`.